frontend/src/components/SplashPage.js

```diff
@@ -1,13 +1,12 @@
 import React from 'react';
+
 import '../styles/SplashPage.css';
-import { getAuth, signInWithPopup, GoogleAuthProvider } from 'firebase/auth';
+import { getAuth, signInWithPopup, GoogleAuthProvider, GithubAuthProvider, createUserWithEmailAndPassword } from 'firebase/auth';
```

Comment:
Are we going to allow creation with email and password? Wasn't sure we could handle that

Reply:
We can remove that imported class. Was just experimenting with forms when I saw that Facebook, Twitter, and Microsoft authentication was a bit more complex than Google, GitHub. Originally wanted 3 creation techniques prior to our Friday meeting last week.